src/App.tsx

```diff
@@ -1,73 +1,91 @@
-import React from 'react';
+import React, { useCallback, useMemo, useRef, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { DropdownMenu } from './components/DropdawnMenu/DropdawnMenu';
```

Comment:
There is a typo in the import path for `DropdownMenu`. It should be `./components/DropdownMenu/DropdownMenu` instead of `./components/DropdawnMenu/DropdawnMenu`.